app/Http/Controllers/Service/CentreController.php

```diff
@@ -127,6 +127,7 @@ public function exportRegistrationsSummary()
 
             // Set the last dates.
             $row["Due Date"] = $due_date;
+            $row["Join Date"] = $reg->family->created_at->format('d/m/Y');
```

Comment:
I mean sure - it should be there if there is a family it will have a created_at but would it be 'good' to `$reg->family->created_at ? $reg->family->created_at->format('d/m/Y') : null;` or even `$reg->family ? $reg->family->created_at->format('d/m/Y') : null;`

To avoid - trying to get of undefined errors in unpredictable situations?

Reply:
Good point. I think i was like "there must always be a family or something is broken". I'm tempted to let it slide, becuase "time!" and then ix it in testing. Mostly because there is no test for this, and should be, so we can bug/test then. ahem.

Reply:
:+1: